src/probes/journal.c

```diff
@@ -122,6 +122,16 @@ static int read_new_entries(sd_journal *journal)
 
                 add_to_payload(data, length);
 
+                // For now, we send one record per log message, in case the
+                // there is a large backlog of messages and we exceed the
+                // payload size limit (8KB). And ignore errors, hoping that it's
+                // a transient problem.
```

Comment:
Ah yea that should work just as well. Drinking from the firehouse of the full log will always be a little tricky.